docs/database/readme.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
+Carts <|-- Status
+Products <|-- Carts
+Users <|-- Addresses
+Users <|-- Carts
+Carts <|-- CartsProducts
+Products <|-- CartsProducts
+Products <|-- ProductImages
+Materials <|-- ProductsMaterials
+Products <|-- ProductsMaterials
+
+class Users {
+    id increments notNull
+    fullName text notNull
+    email text unique notNull
+    isAdmin boolean default false notNull
+    isActive boolean default false notNull
+}
+
+class Categories {
+    id increments notNull
+    name text notNull
+    thumbnailUrl text notNull
+    slug text unique notNull
+    orderOfDisplay number notNull
+}
+
+class Products {
+    id increments notNull
+    name text notNull
+    slug unique notNull
+    categoryId number notNull
+    price number notNull
+    description text notNull
+    thumbnailUrl text notNull
+    outOfStock boolean default false notNull
+    isHighlanderOfTheMoment boolean default false notNull
+}
+
+class ProductImages {
+    id increments notNull
+    productId number notNull
+    imageUrl text notNull
+}
+
+class Materials{
+    id increments notNull
+    name text notNull
+}
+
+class ProductsMaterials {
+    id increments notNull
+    productId number notNull
+    materialId number notNull
+}
+
+class Addresses {
+    id increments notNull
+    userId number notNull
+    address text notNull
+    address2 text
+    city text notNull
+    state text notNull
+    zipCode text notNull
+    country text notNull
+    firstName text notNull
+    lastName text notNull
+    phoneNumber text notNull
+    comment text
+}
+
+class Carts {
+    id increments notNull
+    userId number notNull
+    statusId number notNull
+    shippingAddressId number notNull
```

Comment:
Why do we take in a shippingAddressId instead of a shippingAddress which is a string ?